config/gazebo9_debian_stretch.upstream.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo9_import_debian_stretch
+method: http://packages.osrfoundation.org/gazebo/debian-stable
+suites: [stretch]
+component: main
+architectures: [amd64, arm64, armhf, source]
```

Comment:
if we add armhf here, should we also add i386 ?

Reply:
That's a good question. I'd say we should bring in all arches that are available. Here and on Artful. It will help anyone who wants to build from source.

Reply:
I see this PR has been merged, shoud this be addressed in a follow up then ?

Reply:
Yeah I was just about to comment back. I figured I'd get this merged and we could come back for i386 if there was demand for it.